packages/gatsby/src/bootstrap/load-themes/__tests__/index.ts

```diff
@@ -1,5 +1,5 @@
-const loadThemes = require(`..`)
-const path = require(`path`)
+import loadThemes from "../"
+import * as path from "path"
```

Comment:
We have esModuleInterop enabled, so these can be imported without the star

Reply:
Hmmm let me try.

Reply:
Works!